chaincode/deploy/001_deploy_assetManager.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable node/no-unpublished-import */
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { ethers, upgrades } from "hardhat";
+
+const config = require("../config.js");
+
+const INITIAL_USER_LIMIT = ethers.utils.parseEther("1000000");
+const INITIAL_AUTH_LIMIT = ethers.utils.parseEther("100000");
+
+const deployFunction: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+) {
+  console.log("operators:", JSON.stringify(config.operators));
+
+  const AssetManager = await ethers.getContractFactory("AssetManager");
+  const assetManager = await upgrades.deployProxy(
+    AssetManager,
+    [
+      config.initialOperatorThreshold,
+      config.operators,
+      INITIAL_USER_LIMIT,
+      INITIAL_AUTH_LIMIT,
+    ],
+    { initializer: "initialize", unsafeAllow: ["external-library-linking"] }
+  );
+
+  await assetManager.deployed();
+  console.log("AssetManager deployed to:", assetManager.address);
+  console.log(
+    "AssetManager Operator Threshold:",
+    await assetManager.operatorThreshold()
+  );
+
+  // TODO Display operators on deploy
```

Comment:
What's the issue here?

Reply:
Issue was that in order to be able to retrieve the operators we needed to make access control enumerable in `AssetManager.sol`
i.e  `import "@openzeppelin/contracts-upgradeable/access/AccessControlEnumerableUpgradeable.sol";`
instead of 
``import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";`
Fixed now